src/Mvc/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.targets

```diff
@@ -16,15 +16,24 @@
   </ItemGroup>
 
   <Target Name="OpenApiGetDocuments" Returns="@(_OpenApiProjectDocuments)">
+    <Error Text="OpenAPI document generation is disabled. Add '<OpenApiGenerateDocuments>true</OpenApiGenerateDocuments>' to the project."
```

Comment:
Nice. Didn't know MSBuild handled `<` , `>` inside of a string literal.

Reply:
Not an XML special character when quoted